indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -124,7 +124,9 @@ public TaskStatus call()
                         final File attemptDir = new File(taskDir, attemptUUID);
 
                         final ProcessHolder processHolder;
-                        final int childPort = portFinder.findUnusedPort();
+                        final int childQueryPort = portFinder.findUnusedPort();
```

Comment:
not sure if it's worth it, but I wonder if it would be useful for debugging to keep both ports consecutive, e.g. use the even ports for the main jetty thread, and the odd ports for ingestion, or something like that. Anyone have thoughts on this?